plenoptic/tools/data.py

```diff
@@ -134,63 +144,69 @@ def convert_float_to_int(im, dtype=np.uint8):
 
     """
     if im.max() > 1:
-        raise Exception(f"all values of im must lie between 0 and 1, but max is {im.max()}")
+        raise Exception("all values of im must lie between 0 and 1,"
+                        f"but max is {im.max()}")
     return (im * np.iinfo(dtype).max).astype(dtype)
 
 
 def torch_complex_to_numpy(x):
-    r""" convert a torch complex tensor (written as two stacked real and imaginary tensors)
-    to a numpy complex array
+    r""" convert a torch complex tensor (written as two stacked real
+     and imaginary tensors) to a numpy complex array
 
     Parameters
     ----------------------
     x: `torch.Tensor`
-        Tensor whose last dimension is size 2 where first component is the real component and the second is the
-        imaginary component.
+        Tensor whose last dimension is size 2 where first component is the
+        real component and the second is the imaginary component.
     """
 
     x_np = to_numpy(x)
     if x.ndim not in [5, 6]:
-        raise Exception(f"x has {x.ndim} dimensions, but a complex tensor should have 5 "
-                        "(real and imaginary stacked along the final dim) or 6 if it's a video!")
+        raise Exception(f"x has {x.ndim} dimensions, but a complex tensor"
+                        "should have 5 (real and imaginary stacked along"
+                        "the final dim) or 6 if it's a video!")
     x_np = x_np[..., 0] + 1j * x_np[..., 1]
     return x_np
 
 
-
-def make_basic_stimuli(size=256, requires_grad=True):
+def make_synthetic_stimuli(size=256, requires_grad=True):
     r""" Make a set of basic stimuli, useful for developping and debugging models
 
     Parameters
     ----------
-    size: `int` in [8, 16, 32, 64, 128, 256]
+    size: `int`
         the stimuli will have `torch.Size([size, size])`
     requires_grad: `bool`
         weather to initialize the simuli with gradients
 
     Returns
     -------
-    stimuli: `torch.FloatTensor` of shape [15, 1, size, size]
+    stimuli: `torch.FloatTensor` of shape [B, 1, size, size]
```

Comment:
why is this `B` instead of 15 now? We always return the same stimuli, right?

Reply:
agreed, should be 11 now